HW1/P6/P6.py

```diff
@@ -0,0 +1,101 @@
+import matplotlib
+import numpy as np
+import matplotlib.pyplot as plt
+import multiprocessing as mp
+import seaborn as sns
+sns.set_context('poster', font_scale=1.25)
+from random import randint
+
+# initializing spark
+import pyspark as ps
+
+config = ps.SparkConf()
+config = config.setMaster('local[' + str(2*mp.cpu_count()) + ']')
+config = config.setAppName('P6')
+
+sc = ps.SparkContext(conf=config)
+# REMOVE ALL OF THE DIFFERENT SPARK WARNINGS
+sc.setLogLevel('WARN')
+
+# IMPORT THE TEXT FILE CONTAINING THE PLAY. 
+dataSet=sc.textFile('Shakespeare.txt')
+
+# CLEAN THE DATA AND REMOVE UNWANTED CHARACTER TYPES AS WELL AS DIGITTS AND SPLIT EACH 
+# SENTENCE INTO DISCRETE WORDS. THIS CLEANS UP THE WORDS FROM THE TEXTFILE
+def cleanData(x):
+    x=x.split()
+    words=[]
```

Comment:
I think you can just operate on the entire string directly when checking for uppercase, digits etc.